project1/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Nunito+Sans|Playfair+Display|Titillium+Web&display=swap"
+        rel="stylesheet">
+    <title>project1_The Crane Wife</title>
+</head>
+
+
+
+<body>
+    <header>
+
+        <h1>The Crane Wife</h1>
+        <h2><cite>--by CJ Hauser</h2></cite>
+        <h3>July 16, 2019</h3>
+    </header>
+    <p>Ten days after I called off my engagement I was supposed to go on a scientific expedition to study the whooping
+        crane on the gulf coast of Texas. <span class="thought"><i>Surely, I will cancel this trip</i></span>, I
```

Comment:
You don't need the `span` and the `i` here; you can add the class to the `i` element:
`<i class="thought">`
You'd use the `span` if there _wasn't`_ already another element here to add your class to.